docs/sources/guides/whats-new-in-v6-3.md

```diff
@@ -0,0 +1,127 @@
+type = "docs"
+[menu.docs]
+name = "Version 6.3"
+identifier = "v6.3"
+parent = "whatsnew"
+weight = -14
++++
+
+# What's New in Grafana v6.3
+
+For all details please read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md)
+
+## Highlights
+
+- New Explore features
+  - [Loki Live Streaming]({{< relref "#loki-live-streaming" >}})
+  - [Loki Context Queries]({{< relref "#loki-context-queries" >}})
+  - [Elasticsearch Logs Support]({{< relref "#elasticsearch-logs-support" >}})
+  - [InfluxDB Logs Support]({{< relref "#influxdb-logs-support" >}})
+- [Data links]({{< relref "#data-links" >}})
+- [New Time Picker]({{< relref "#new-time-picker" >}})
+- [Graph Area Gradients]({{< relref "#graph-gradients" >}}) - A new graph display option!
+- Grafana Enterprise
+  - [LDAP Active Sync]({{< relref "#ldap-active-sync" >}}) - LDAP Active Sync
+  - [SAML Authentication]({{< relref "#saml-authentication" >}}) - SAML Authentication
+
+## Explore improvements
+
+This release adds a ton of enhancements to Explore. Both in terms of new general enhancements but also in
+new data source specific features.
+
+### Loki live streaming
+
+For log queries using the Loki data source you can now stream logs live directly to the Explore UI.
+
+### Loki context queries
+
+After finding a log line through the heavy use of query filters it can then be useful to
+see the log lines surrounding the line your searched for. The `show context` feature
+allows you to view lines before and after the line of interest.
+
+### Elasticsearch logs support
+
+This release adds support for searching & visualizing logs stored in Elasticsearch in the Explore mode. With a special
+simplified query interface specifically designed for logs search.
+
+{{< docs-imagebox img="/img/docs/v63/elasticsearch_explore_logs.png" max-width="600px" caption="New Time Picker" >}}
+
+### InfluxDB logs support
+
+This release adds support for searching & visualizing logs stored in InfluxDB in the Explore mode. With a special
+simplified query interface specifically designed for logs search.
+
+{{< docs-imagebox img="/img/docs/v63/influxdb_explore_logs.png" max-width="600px" caption="New Time Picker" >}}
+
+## Data Links
+
+We have simplified the UI for defining panel drilldown links (and renamed them to Panel links). We have also added a
+new type of link named `Data link`. The reason to have two different types is to make it clear how they are used
+and what variables you can use in the link. Panel links are only shown in the top left corner of
+the panel and you cannot reference series name or any data field.
+
+While `Data links` are used by the actual visualization and can reference data fields.
+
+Example:
+```url
+http://my-grafana.com/d/bPCI6VSZz/other-dashboard?var-server=${__series_name}
+```
+
+You have access to these variables:
+
+Name | Description
+------------ | -------------
+*${__series_name}* | The name of the time series (or table)
```

Comment:
For me, it looks kinda weird (am I doing something wrong?) - 
<img width="929" alt="Screenshot 2019-07-10 at 12 22 08" src="https://user-images.githubusercontent.com/945528/60957528-78513100-a30d-11e9-8f8e-c1edd942485a.png">

Reply:
Yes, old docs-base image , need to rebuild it (from old-docs branch)